### PR TITLE
use google sheet api for contact us and signup

### PR DESCRIPTION
**Pull request by cheedinmar**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+VUE_APP_URL_CONTACT_PRODUCTION="https://script.google.com/macros/s/AKfycbwrJJ7a5neSbIb4b0S6NQpLq27ev82BgO14cb3V4a0xjkT58eM4iHxyOPuRMAOBFEQa/exec"
```
Comment (teebarg) [inline .env:1]: add .env file to git ignore

Comment (cheedinmar) [reply .env:1]: ok

File: src/components/HomePage/ContactUs.vue [hunk 2]
```diff
@@ -104,7 +104,8 @@ export default {
       phone_number: "",
       preferred_method_of_communication: "",
       message: "",
-      form_sent: false
+      form_sent: false,
+      formURL: process.env.NODE_ENV == "development" ? process.env.VUE_APP_URL_CONTACT_STAGING : process.env.VUE_APP_URL_CONTACT_PRODUCTION
```
Comment (teebarg) [inline src/components/HomePage/ContactUs.vue:108]: we don't need this check, just set the one variable, don't set either for staging or prod

File: src/components/SignUp.vue [hunk 3]
```diff
@@ -135,7 +135,8 @@ export default {
       phone_number: "",
       services_you_are_interested_in: "",
       employment_status: "",
-      form_sent: false
+      form_sent: false,
+      formURL: process.env.NODE_ENV == "development" ? process.env.VUE_APP_URL_SIGNUP_STAGING : process.env.VUE_APP_URL_SIGNUP_PRODUCTION
```
Comment (teebarg) [inline src/components/SignUp.vue:139]: Same here

File: src/components/HomePage/ContactUs.vue (hunk 2 above)
Comment (cheedinmar) [reply src/components/HomePage/ContactUs.vue:108]: ok

File: src/services/apiService.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+    try {
+      let result = await this.api_connector.post(
+        url,
+        postData,
+        this.requestConfig
+      );
+      this.resetRequestConfig();
+
+      console.log("success", result);
```
Comment (teebarg) [inline src/services/apiService.js:66]: Remove log

File: src/services/apiService.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+      this.requestConfig.headers = header;
+    }
+
+    return this;
+  }
+  async get(url) {
+    try {
+      let result = await this.api_connector.get(url, this.requestConfig);
+      this.resetRequestConfig();
+      console.log(result.data);
```
Comment (teebarg) [inline src/services/apiService.js:37]: remove log

File: src/components/HomePage/ContactUs.vue [hunk 6]
```diff
@@ -104,7 +104,8 @@ export default {
       phone_number: "",
       preferred_method_of_communication: "",
       message: "",
-      form_sent: false
+      form_sent: false,
+      formURL: process.env.VUE_APP_URL_CONTACT_STAGING
```
Comment (teebarg) [inline src/components/HomePage/ContactUs.vue:108]: Change to `VUE_APP_URL_CONTACT`

File: src/components/SignUp.vue [hunk 7]
```diff
@@ -135,7 +135,8 @@ export default {
       phone_number: "",
       services_you_are_interested_in: "",
       employment_status: "",
-      form_sent: false
+      form_sent: false,
+      formURL: process.env.VUE_APP_URL_SIGNUP_STAGING
```
Comment (teebarg) [inline src/components/SignUp.vue:139]: change to `VUE_APP_URL_SIGNUP`

File: .env [hunk 8]
```diff
@@ -0,0 +1,5 @@
...
+VUE_APP_URL_CONTACT_STAGING="https://script.google.com/macros/s/AKfycbz_agnXpA70-PH6JYaF_PklZfBjlV-1-AhtGtnHeu6PUurOuC3KJWREgvrN6Yriu_Zv/exec"
```
Comment (teebarg) [inline .env:2]: why is this file still here?

**Review by teebarg — APPROVED** [review]

**Pull request merged**